addons/mail/tests/test_mail_activity.py

```diff
@@ -18,3 +19,69 @@ def test_chatter_activity_tour(self):
             "mail_activity_schedule_from_chatter",
             login="admin",
         )
+
+
+class TestActivityScheduleCommon(HttpCase):
```

Comment:
We now try to use "...Case" for tools classes, to distinguish them from classes holding unit tests, like
```suggestion
class ActivityScheduleCase(HttpCase):
```